switchbot_ros/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(switchbot_ros)
+
+find_package(catkin REQUIRED COMPONENTS
+  catkin_virtualenv
```

Comment:
do  you really need virtualenv? does python3 necessary, because of urllib? if so, how about https://stackoverflow.com/questions/6594620/python-3-2-unable-to-import-urllib2-importerror-no-module-named-urllib2